src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1912,12 +1912,12 @@ fsal_status_t kvsfs_extract_handle(struct fsal_export *exp_hdl,
 
 	assert(fh_desc);
 
-	if (in_type != FSAL_DIGEST_NFSV4) {
+	/*if (in_type != FSAL_DIGEST_NFSV4) {
```

Comment:
Instead of commenting, just remove it. Keep a comment stating that we accept both kind of digests